doc/interop-test-descriptions.md

```diff
@@ -679,6 +679,75 @@ Client asserts:
 by the auth library. The client can optionally check the username matches the
 email address in the key file.
 
+### google_default_credentials_no_key_file
+
+Similar to the other auth tests, this test is only for cloud-to-prod path.
+
+This test verifies unary calls succeed with GoogleDefaultCredentials
+when the path to a service account key file _is not_ provided by the
+GOOGLE_APPLICATION_CREDENTIALS environment variable and the client is run
+on GCP (this test also assumes that user credentials under
+`$HOME/.config/gcloud/application_default_credentials.json` are _not_
+accessible by the client under test).
+
+The test uses `--default_service_account` with GCE service account email and
+`--oauth_scope` with the OAuth scope to use. For testing against
+grpc-test.sandbox.googleapis.com, "https://www.googleapis.com/auth/xapi.zoo"
+should be passed in as `--oauth_scope`.
+
+Server features:
+* [UnaryCall][]
+* [Echo Authenticated Username][]
+* [Echo OAuth Scope][]
+
+Procedure:
+ 1. Client configures channel to use GoogleDefaultCredentials
+ 2. Client calls Unarychannel with:
+
+    ```
+    {
+      fill_username: true
+      fill_oauth_scope: true
+    }
+    ```
+
+Client asserts:
+* call was successful
+* received SimpleResponse.username equals the value of
+  `--default_service_account` flag
+* received SimpleResponse.oauth_scope matches the value of `--oauth_scope`
+
+### google_default_credentials_key_file_provided
```

Comment:
Discussed offline. We're going to go with a single google_default_creds test. The environment that it runs it can be configured in different ways to test different scenarios.